src/index.d.ts

```diff
@@ -38,8 +38,8 @@ declare namespace dynamodb {
   export function tableSchema(
     schema: ReadonlyArray<aws.DynamoDB.Types.CreateTableInput>
   ): void;
-  export function dynamoDBTestHooks(useUniqueTables?: boolean): Hooks;
-  export function useDynamoDB(test: TestInterface, useUniqueTables?: boolean): void;
+  export function dynamoDBTestHooks(useUniqueTables?: boolean, {inMemory}?: {inMemory: boolean}): Hooks;
```

Comment:
Does the `{inMemory}?: {inMemory: boolean}` syntax work? I haven't seen that before. I expected to see `args?: {inMemory: boolean}`